schema_source/jetstream/api/v1/definitions.json

```diff
@@ -84,7 +88,15 @@
       "properties": {
         "name": {
           "type": "string",
-          "description": "The name of the Stream being replicated"
+          "description": "The name of the Stream being sourced"
```

Comment:
`replicated` is definitely better than `sourced` here since this structure is also used in the context of Mirrors where `sourced` is confusing.

Reply:
Sure, I had changed it because (in my mind) I see mirror as just another kind of source (that happens to be exclusive, etc...) and 'replicated' (again, for me) I associate just with replicating the messages between servers, but no worries changed it back to 'replicated' now.

Reply:
It's also called replication in docs. https://docs.nats.io/running-a-nats-service/nats_admin/jetstream_admin/replication